pkg/helm/helm.go

```diff
@@ -87,12 +85,6 @@ func (h *HelmWrapper) Pull(logger logr.Logger, url string) (chartRequested *char
 	entryName := combinedName[len(combinedName)-1]
 
 	i := action.NewPullWithOpts(action.WithConfig(h.config))
-	h.config.RegistryClient, err = registry.NewClient()
```

Comment:
https://github.com/kubebb/core/blob/80225051d352ff190eb7cea49bdf8219fc01264b/pkg/helm/helm.go#L68-L76 has define it.